src/depth-to-rgb-calibration.cpp

```diff
@@ -35,6 +36,12 @@ depth_to_rgb_calibration::depth_to_rgb_calibration(
     auto yuy_data = (impl::yuy_t const *) yuy.get_data();
     auto prev_yuy_data = (impl::yuy_t const *) prev_yuy.get_data();
     impl::calib calibration( _intr, _extr );
+
+    if (!should_continue())
+    {
+        throw librealsense::camera_disconnected_exception("The device has been disconnected during Camera Accuracy Health process");
```

Comment:
I'd rather the function threw the exception if you want to make the text this verbose: the code here does not know, and should not assume to know, why we don't want to continue.

Reply:
Agreed, Will move the exception to the function